scout/constants/query_terms.py

```diff
@@ -62,4 +62,5 @@
     "split_reads",
     "fusion_caller",
     "rank_score",
+    "clinsig_onc",
```

Comment:
Why not primary?

Reply:
It started as a primary. But after spending one day on the code, which  became a horrible mess, I opted to make it secondary, since it is much easier to maintain.  If it is primary, you need to take care of the fact that there can be:

- ClinVar criteria
- Exclusion of Clinvar criteria
- Oncogenicity criteria
- Exclusioon of the oncogenocity criteria
- Show only vars with ClinVar criteria
- Show only vars with Clinsig confident

All of these in different combinations. We can of course have it as a primary, but I would rather refactor and simplify the code first. Also cancer variants don't have the clinical filter thing with the terms in $or, so I figured this was kind of the expected behavior.

Reply:
Thats ok. I still think once there are enough calls in the ClinVar ONC collection, we will want to be able to use it in an inclusive fashion as well, much as the germline ones, to make sure one is not missing important variants.